tuxedo-core/src/verifier/htlc.rs

```diff
@@ -0,0 +1,380 @@
+//! This module contains `Verifier` implementations related to Hash Time Lock Contracts.
```

Comment:
For review purposes, this file is the main additions.

The rest of the diff is mostly moving existing verifiers to dedicated files.